src/ResourceManager/Network/Commands.Network/help/Add-AzureRmRouteFilterRuleConfig.md

```diff
@@ -35,6 +35,10 @@ The Add-AzureRmRouteFilterRuleConfig cmdlet adds a route filter rule to an Azure
 
 
 
+
```

Comment:
@EvgenyAgafonchikov please make sure the added blank lines in markdown files are removed